LocationServices/LocationServices/Scenes/About/Controller/AboutVC.swift

```diff
@@ -11,8 +11,7 @@ final class AboutVC: UIViewController {
     
     private var screenTitleLabel: UILabel = {
```

Comment:
create a custom view for titleLabel and reuse it everywhere

Reply:
fixed